### DISCUSSION TITLE
SDL3 Audio changed a lot

### DISCUSSION THREAD

**Discussion by felipetesc** [revmischa/sdl3-rs#13] [General]
When I compiled this crate, I found many errors on the audio crate, and mostly because SDL3 changed the API a lot: 

> The audio subsystem in SDL3 is dramatically different than SDL2. The primary way to play audio is no longer an audio callback; instead you bind SDL_AudioStreams to devices; however, there is still a callback method available if needed.

More about that at https://github.com/libsdl-org/SDL/blob/main/docs/README-migration.md

**Comment by revmischa** [comment]
Yep I was working on that here: https://github.com/revmischa/sdl3-rs/pull/7
Feel free to pick it up and continue the work if you have time. I don't for the next couple months at least.

**Reply by felipetesc** [reply]
I though about asking them (SDL's Team) to create tags or the sort but:
At https://github.com/libsdl-org/SDL/issues/10367:
**_Because the API is changing daily and we don't want people to freeze on a specific version. We have an ABI stable release coming soon, and we'll tag that for people to play with._**
